CMakeLists.txt

```
@@ -225,7 +237,7 @@ add_subdirectory(examples)
 #============================================================================
 ign_create_packages()
 
-if (${pybind11_FOUND})
+if (pybind11_FOUND AND NOT SKIP_PYBIND11)
```

For `pybind11_FOUND` to be true it needs that `NOT SKIP_PYBIND11` would be also true given that the `find_package` command [is conditioned to it](https://github.com/gazebosim/gz-sim/pull/2005/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR209-R213).  Does this work if we leave the code just:
```suggestion
if (pybind11_FOUND)
```